randovania/games/samus_returns/logic_database/Area 1.txt

```diff
@@ -1039,6 +1039,12 @@ Extra - asset_id: collision_camera_042
       Trivial
   > Tunnel to Teleporter Room
       Lay Any Bomb
+  > Dock to Teleporter Room
+      Morph Ball and Melee Clip (Intermediate) and Fully Freeze Enemy
+
+> Dock to Teleporter Room; Heals? False
+  * Layers: default
+  * Open Passage to Teleporter Room/Teleporter
```

Comment:
You need to add a dock in the teleporter room as well, which connects to the teleporter or right dock, and then connect both new docks together